### PR TITLE
fix: [#72] Adds translation on the location views

### PR DESCRIPTION
**Pull request by andrebonon**

### CODE REVIEW

**Comment by podarok** [comment]
@andrebonon we should write hook_update_N and update/install configs/config parameters for existing sites during upgrade

**Comment by andrebonon** [comment]
@podarok please check the last addition.

File: openy_map.install [hunk 1]
```diff
@@ -148,9 +148,19 @@ function openy_map_update_8008() {
...
 }
+
+/**
+ * Adds default language condition to the view location.
+ */
+function openy_map_update_8010() {
+  $config_path = \Drupal::service('extension.path.resolver')->getPath('module', 'openy_map') . '/config/install';
+  $source = new FileStorage($config_path);
+  $config_storage = \Drupal::service('config.storage');
+  $config_storage->write('views.view.locations', $source->read('views.view.locations'));
```
Comment (podarok) [inline openy_map.install:165]: please, use config updater @andrebonon https://github.com/open-y-subprojects/openy_map/pull/74/files#diff-f5f5f7f0a35898108deac0ddd5770f86a2910ef5c8f0be5ac7fb7b499d5d6804R118

config write not always run proper subscriber, and not all of the,m

**Review by podarok — CHANGES_REQUESTED** [review]
needs https://github.com/open-y-subprojects/openy_map/pull/74/files#diff-f5f5f7f0a35898108deac0ddd5770f86a2910ef5c8f0be5ac7fb7b499d5d6804R118 instead of config write, please

File: openy_map.info.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ description: Renders a map with locations. Provides blocks for Locations page.
 core_version_requirement: ^8 || ^9 || ^10
 version: 1.15
```
Comment (podarok) [inline openy_map.info.yml:5]:
```suggestion
version: 5.1.1
```

**Pull request merged**